ui/perfherder/tests/TestsTable.jsx

```diff
@@ -31,77 +38,159 @@ export default function TestsTable(props) {
-    },
-    {
-      headerStyle,
-      Header: 'Projects',
-      accessor: 'repositories',
-      Cell: (props) => {
-        if (projectsMap) {
-          const repositories = props.value.map((id) => projectsMap[id]);
-          return <ItemList items={repositories} />;
-        }
-        return null;
-      },
-      width: 300,
+      columns: [
+        {
+          headerStyle,
+          Header: 'Suite',
+          accessor: 'suite',
+          Cell: ({ row }) => {
+            const perfdocs = new Perfdocs(framework, row.suite);
+            const hasDocumentation = perfdocs.hasDocumentation(
+              perfViews.testsView,
+            );
+            return (
+              <div>
+                {hasDocumentation ? (
+                  <a
+                    href={perfdocs.documentationURL}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {row.suite}
+                  </a>
+                ) : (
+                  <div>{row.suite}</div>
+                )}
+              </div>
+            );
+          },
+        },
+        {
+          headerStyle,
+          Header: 'Test Name',
+          accessor: 'test',
+        },
+        {
+          headerStyle,
+          Header: 'Platforms',
+          accessor: 'platforms',
+          Cell: (props) => {
+            if (platformsMap) {
+              const platforms = props.value.map((id) => platformsMap[id]);
+              return <PlatformList items={platforms} />;
+            }
+            return null;
+          },
+          width: 300,
+          style: { textAlign: 'center' },
+          sortable: false,
+        },
+        {
+          headerStyle,
+          Header: 'Projects',
+          accessor: 'repositories',
+          Cell: (props) => {
+            if (projectsMap) {
+              const repositories = props.value.map((id) => projectsMap[id]);
+              return <ItemList items={repositories} />;
+            }
+            return null;
+          },
+          width: 300,
+          style: { textAlign: 'center' },
+          sortable: false,
+        },
+      ],
+      width: 400,
       style: { textAlign: 'center' },
-      sortable: false,
     },
     {
       headerStyle,
       Header: 'Alerts',
-      accessor: 'total_alerts',
-      Cell: (props) => {
-        const { original } = props;
-        return (
-          <AlertsLink
-            alerts={original}
-            framework={framework}
-            allFrameworks={allFrameworks}
-          />
-        );
-      },
+      columns: [
+        {
+          headerStyle,
+          Header: () => (
+            <SimpleTooltip
+              tooltipText="Improvements"
+              text={
+                <FontAwesomeIcon
+                  icon={faArrowAltCircleUp}
+                  className="text-success"
+                />
+              }
+            />
+          ),
+          accessor: 'total_alerts',
```

Comment:
What is the purpose of `accessor: 'total_alerts'`?

Reply:
This seems to be a [required field](https://react-table.tanstack.com/docs/api/useTable#column-options), it is used to give the column an id and for sorting purpose. I think I can change the accessor for `Regressions` and `Untriaged` columns with `total_regressions` and `total_untriaged`. I don't know what to set for the `Improvements` column. If you have any suggestion, let me know. There is no accessor as `total_improvements`.

For now the sorting method for these 3 columns is off. Do you think we should enable it?

Reply:
Could you use `accessor: 'improvements'` for the `Improvements` column? Since all improvements are kept in the `improvements` constant?